service-catalog/generic-service.yaml

```diff
@@ -0,0 +1,45 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  annotations:
+    velocity.tech.v1/id: generic-service-var-service-id
+  name: generic-service-var-service-id
+spec:
+  replicas: 1
+  revisionHistoryLimit: 10
```

Comment:
redundant delete as well 🙏

Reply:
sorry, I didn't get, what exactly is redundant here. Do you mean replicas and revisionHistoryLimit?